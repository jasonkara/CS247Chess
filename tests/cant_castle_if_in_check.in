game human human
move e2 e4
move d7 d5
move e4 d5
move d8 d5
move g1 f3
move a7 a6
move f1 d3
move a6 a5
move a2 a3
move d5 e5
move e1 g1
