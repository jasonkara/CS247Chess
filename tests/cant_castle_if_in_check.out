8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _p_ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr
7 ppp pppp
6  _ _ _ _
5 _ _P_ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh
8 rnb_kbnr
7 ppp pppp
6  _ _ _ _
5 _ _q_ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh
8 rnb_kbnr
7 ppp pppp
6  _ _ _ _
5 _ _q_ _ 
4  _ _ _ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQKB_R

  abcdefgh
8 rnb_kbnr
7 _pp pppp
6 p_ _ _ _
5 _ _q_ _ 
4  _ _ _ _
3 _ _ _N_ 
2 PPPP PPP
1 RNBQKB_R

  abcdefgh
8 rnb_kbnr
7 _pp pppp
6 p_ _ _ _
5 _ _q_ _ 
4  _ _ _ _
3 _ _B_N_ 
2 PPPP PPP
1 RNBQK _R

  abcdefgh
8 rnb_kbnr
7 _pp pppp
6  _ _ _ _
5 p _q_ _ 
4  _ _ _ _
3 _ _B_N_ 
2 PPPP PPP
1 RNBQK _R

  abcdefgh
8 rnb_kbnr
7 _pp pppp
6  _ _ _ _
5 p _q_ _ 
4  _ _ _ _
3 P _B_N_ 
2  PPP PPP
1 RNBQK _R

  abcdefgh
8 rnb_kbnr
7 _pp pppp
6  _ _ _ _
5 p _ q _ 
4  _ _ _ _
3 P _B_N_ 
2  PPP PPP
1 RNBQK _R

  abcdefgh
White is in check.
8 rnb_kbnr
7 _pp pppp
6  _ _ _ _
5 p _ q _ 
4  _ _ _ _
3 P _B_N_ 
2  PPP PPP
1 RNBQK _R

  abcdefgh
White is in check.
Final Score:
White: 0
Black: 0
